packages/renderer/src/lib/pod/DeployPodToKube.svelte

```diff
@@ -22,6 +22,7 @@ let deployError = '';
 let deployWarning = '';
 let updatePodInterval: NodeJS.Timeout;
 let openshiftConsoleURL: string;
+let routeGroupSupported = false;
```

Comment:
```suggestion
let openshiftRouteGroupSupported = false;
```

probably says that it's openshift route ?